packages/cli/src/commands/doctor/healthchecks/reactNativeGlobal.ts

```diff
@@ -26,42 +30,48 @@ const findGlobalPackage = (packageManager, moduleId) => {
   }
 };
 
-const checkGlobalPaths = moduleId => {
-  return !!(
-    (findGlobalPackage(packageManagers.yarn, moduleId) ||
-      findGlobalPackage(packageManagers.npm, moduleId)) &&
-    true
-  );
+const checkGlobalInstallalations = () => {
+  //Returns an array of globally installed paths for rn and rncli both in yarn and npm
+  const librariesGloballyInstalled = Object.keys(packages)
+    .map(packageKey =>
+      Object.keys(packageManagers).map(packageManagerKey =>
+        findGlobalPackage(
+          packageManagers[packageManagerKey],
+          packages[packageKey],
+        ),
+      ),
+    )
+    .filter(e => e != null);
+
+  //Cant's use flat() in ts...? This is an alternative
+  const flattenedArray = [].concat.apply([], librariesGloballyInstalled);
```

Comment:
Why is this needed?

Reply:
I included this because mapping over the packages/packageManagers returns a nested array that needs to be flattened? When you console.log this nested array you can't see it for some reason in the terminal (not sure why?), but look in the console on this playground (you also can't see it in the playground display area):

<img width="1314" alt="image" src="https://user-images.githubusercontent.com/20524533/75756547-bbdb2e00-5d28-11ea-8b4d-d3b07720c0b1.png">

Reply:
It's because the terminal does not expand further.

To log everything you can use `JSON.stringify(variable);`.